src/AppBundle/Resources/config/routing/routing_deck.yml

```diff
@@ -112,6 +112,24 @@ deck_export_text:
     requirements:
         deck_id: \d+
 
+deck_export_text_ordered_deluxe_before:

```

Comment:
needs equivalent routes defined in `routing_decklist.yaml`.

e.g.

```
decklist_export_text_ordered_deluxe_before:
    path: /export/text_ordered_db/{deck_id}
    methods: [GET]
    defaults:
        _controller: AppBundle:Social:textorderedexport
        deluxeAfter: false
    requirements:
        decklist_id: \d+

decklist_export_text_ordered_deluxe_after:
    path: /export/text_ordered_da/{deck_id}
    methods: [GET]
    defaults:
        _controller: AppBundle:Social:textorderedexport
        deluxeAfter: true
    requirements:
        decklist_id: \d+
```

Reply:
Solved in 6e6c54df53086d6174d428bc51feab923d66d56f